src/QAT/qat/purr/compiler/devices.py

```diff
@@ -494,6 +494,9 @@ def __init__(self, direction, quality=1):
         self.direction = tuple(direction)
         self.quality = 1 if quality < 1 else quality
 
+    def __repr__(self):
```

Comment:
I don't understand its use :thinking:

Reply:
oh absolutely, I was under the impression it had a role in the actual fix itself ^^